src/ui/public/inspector/ui/inspector_panel.js

```diff
@@ -106,15 +115,18 @@ class InspectorPanel extends Component {
 }
 
 InspectorPanel.defaultProps = {
-  title: 'Inspector',
+  title: inspectorTitle,
 };
 
 InspectorPanel.propTypes = {
   adapters: PropTypes.object.isRequired,
   views: (props, propName, componentName) => {
     if (!Array.isArray(props[propName]) || props[propName].length < 1) {
       throw new Error(
-        `${propName} prop must be an array of at least one element in ${componentName}.`
+        i18n.translate('common.ui.inspector.propMustBeArrayError', {
```

Comment:
Prop type validations are only shown on the browsers dev console, so no need to translate this.